src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -130,7 +130,46 @@ public void testLegacyAccountEntityToAttributes() {
         assertEquals(a.getName(), attr.name);
         assertNull(a.getStudentProfile());
         assertNull(attr.studentProfile);
+    }
+
+    @Test
+    public void testGetCopy_typicalData_shouldCreateDeepCopy() {
+        AccountAttributes account = createValidAccountAttributesObject();
+
+        AccountAttributes copy = account.getCopy();
+
+        assertFalse(account.equals(copy));
```

Comment:
There are differences between
```
account.equals(copy) // we usually use to check the equality of object (deep-equal)
```
and 
```
account == copy // we use to check the equality of reference to the object
```
If you use `account.equals(copy)`, it should be true in this case. However, since we didn't override the equals method in `AccountAttribute`, it functions like `account == copy`. In this test case, we want to check that whether `getCopy()` gives us a brand new copy or not and therefore we should check their references. `assertFalse(account == copy)` or `assertTrue(account != copy)` should be used.

Reply:
Can use `assertNotSame`.

Reply:
Ok. I will use it.